src/codegen/expression.rs

```diff
@@ -3586,6 +3586,7 @@ pub(super) fn assert_failure(
 }
 
 /// Generate the binary code for a contract
+#[cfg(feature = "llvm")]
 fn code(loc: &Loc, contract_no: usize, ns: &Namespace, opt: &Options) -> Expression {
```

Comment:
Is it possible to merge this function and the one below like this?
```rust
#[cfg(feature = "llvm")]
let code = contract.emit(ns, opt);

#[cfg(not(feature = "llvm"))]
let code = b"code placeholder".to_vec();

// Rest of the function
```

Reply:
You'll get clippy warnings that `ns`, `opt`, `contract` are unused variables.